starter-code/pizza.js

```diff
@@ -1 +1,131 @@
 // Write your Pizza Builder JavaScript in this file.
+$("document").ready(function () {
+
+    // make crust and red sauce default on load
+    $(".crust").removeClass("crust-gluten-free");
+
+    $(".sauce").removeClass("sauce-white");
+
+    //setting default price on load
+    $(".total").html("13");
+
+
+
+
+    // "if the ingredient is turned on, its button should have active."
+    //removing active class so it will appear again on click. 
+
+    // $(".btn-green-peppers").removeClass("active");
+
+    // $(".btn-mushrooms").removeClass("active");
+
+    // $(".btn-pepperonni").removeClass("active");
+
```

Comment:
Please make sure you do a "clean up" commit before you submit a pull request.  Get rid of all unnecessary comments, extra empty lines (you should typically be using 1 at most) and any other unused variables or functions